lib/mi_bridges/driver.rb

```diff
@@ -49,6 +49,11 @@ class Driver
       HousingUtilityBillsSummaryPage,
       YourOtherBillsExpensesPage,
       YourOtherBillsExpensesSummaryPage,
+      SchoolDetailsPage,
+      DebuggerPage,
```

Comment:
do we want to keep these in here?

Reply:
@jessieay - These are placeholder pages so you can manually step through the entire process. When we backfill the later pages we will remove them.